src/Date/Month.tsx

```diff
@@ -302,18 +303,23 @@ function Month(props: MonthSingleProps | MonthRangeProps | MonthMultiProps) {
             >
               {monthName} {year}
             </Text>
-            <View style={isHorizontal ? styles.opacity1 : styles.opacity0}>
-              <IconButton
-                onPress={isHorizontal ? () => onPressYear(year) : undefined}
-                disabled
-                icon={
-                  selectingYear
-                    ? theme.isV3
-                      ? 'menu-up'
-                      : 'chevron-up'
-                    : theme.isV3
-                    ? 'menu-down'
-                    : 'chevron-down'
+            <View style={{
+              ...(isHorizontal ? styles.opacity1 : styles.opacity0),
```

Comment:
can you make this a style instead of inline styles like [styles.iconWrapper, isHorizontal ? styles.opacity1 : styles.opacity0 ]